### DISCUSSION TITLE
chatgpt sidebar extensiion should get the gx theme

### DISCUSSION THREAD

**Discussion by Midnight0507** [Godiesc/firefox-gx#106] [Ideas]
![image](https://github.com/Godiesc/firefox-gx/assets/110766220/3bb2fb47-008c-46fd-92f3-4983725b087a)
https://addons.mozilla.org/en-US/firefox/addon/chatgpt-sidebar-for-fire-fox/  -- extension link

**Comment by Godiesc** [comment]
hi, I will add the code for the next update, for now you can use this code in you `ogx_tricks.css` file: 
```
#PersonalToolbar #_63ec59d8-ba50-4d88-917e-fb65af892a39_-BAP { 
list-style-image: url("../images/logo_chatgpt.svg") !important; 
}
```

**Reply by Midnight0507** [reply]
got it !!
